test/controllers/users/invitations_controller_test.rb

```diff
@@ -0,0 +1,10 @@
+require 'test_helper'
+
+class Users::InvitationsControllerTest < ActionController::TestCase
+  test "admin can access invite page" do    
+    user = User.create(:email => "example@example.com", :password => "password", :password_confirmation => "password", :admin => true)
+    sign_in user
+    get :new
```

Comment:
This line returns the following error message whenever I run `rake`:

`NoMethodError: undefined method `name' for nil:NilClass`